lib/wibox/layout/margin.lua

```diff
@@ -9,6 +9,7 @@ local pairs = pairs
 local type = type
 local setmetatable = setmetatable
 local base = require("wibox.widget.base")
+local util = require("awful.util")
```

Comment:
unused import and the `recursive` argument below isn't used either (I'll stop making this comment on all instances of this)